cljfmt/src/cljfmt/core.cljc

```diff
@@ -3,6 +3,7 @@
        [(:require
          [clojure.java.io :as io]
          [clojure.zip :as zip]
+         [clojure.string :as str]
```

Comment:
I was in the process of my second interactive rebase having noticed that, should be good now.